docs/audit/2026-05-29/07-docs.md

```
@@ -0,0 +1,82 @@
+# Docs & Developer Experience Audit — 2026-05-29
+
+README and CLAUDE.md disagree on the project's architecture, model, and phase status; README has 18 broken links; CONTRIBUTING tells new devs to use Poetry while the project runs on uv.
+
+## Findings
+
+### DOC-01 — README architecture contradicts CLAUDE.md (model + phase) — High — M
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)